pkgs/games/prismlauncher/default.nix

```diff
@@ -34,23 +41,36 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-RArg60S91YKp1Mt97a5JNfBEOf2cmuX4pK3VAx2WfqM=";
   };
 
+  patches = lib.optional stdenv.isDarwin (fetchpatch {
+    # cherry picked commits that have been merged upstream. allows us to disable the sparkle updater
+    # and fixup_bundle during builds, which will cause failures otherwise
+    # TODO: remove this when updating to 8.0
+    name = "toggle-sparkle-and-bundling.patch";
+    url = "https://patch-diff.githubusercontent.com/raw/PrismLauncher/PrismLauncher/pull/1452.patch";
+    hash = "sha256-A79dTXOnDK44curiUgrPRrMzbdH/PzTX3VSqJd6V7Go=";
+  });
```

Comment:
Using the individual commits instead of the PR should be more reliable long term, even though the PR has already been merged.

```suggestion
  patches = lib.optionals stdenv.isDarwin [
    # https://github.com/PrismLauncher/PrismLauncher/pull/1452
    # These patches allow us to disabled the Sparkle updater and cmake bundling
    # TODO: remove these when updating to 8.0
    (fetchpatch {
      name = "disable-sparkle-when-url-is-empty.patch";
      url = "https://github.com/PrismLauncher/PrismLauncher/commit/48e50401968a72846350c6fbd76cc957b64a6b5a.patch";
      hash = "sha256-IFxp6Sj87ogQcMooV4Ql5/4B+C7oTzEk+4tlMud2OLo=";
    })
    (fetchpatch {
      name = "make-install_bundle-cached.patch";
      url = "https://github.com/PrismLauncher/PrismLauncher/commit/a8498b0dab94d0ab6c9e5cf395e5003db541b749.patch";
      hash = "sha256-ji5GGUnzVut9xFXkynqf9aVR9FO/zsqIbt3P9dexJ2I=";
    })
    (fetchpatch {
      name = "dont-include-sparkle-when-not-enabled.patch";
      url = "https://github.com/PrismLauncher/PrismLauncher/commit/51bfda937d47837ed426150ed6f43a60b4ca0ce1.patch";
      hash = "sha256-7hMgANOg4zRIf3F2AfLXGR3dAEBqVmKm/J5SH0G5oCk=";
    })
    (fetchpatch {
      name = "introduce-internal-updater-variable.patch";
      url = "https://github.com/PrismLauncher/PrismLauncher/commit/b1aa9e584624a0732dd55fc6c459524a8abfe6ba.patch";
      hash = "sha256-mm++EfnBxz7NVtKLMb889mMq8F/OdQmzob8OmlvNlRA=";
    })
  ];
```

Reply:
good idea! just updated it :)